### PR TITLE
Performance of LSTM with or without cuDNN(v5) in Chainer

### PR DESCRIPTION
**Pull request by aonotas**
Original draft was written at here.
https://docs.google.com/document/d/163Lf04JlzqaEKS5adR0jjyyR53JtuP8wE1_7esJ0nbM/edit

Can we replace static image file to iframe tag (Google Spreadsheet graph iframe tag)?

### CODE REVIEW

**Comment by unnonouno** [comment]
If it is easy please use iframe (sorry, i'm not familiar with google sphreadsheet).

**Comment by unnonouno** [comment]
Ah, I think images are safer because API will be changed.

**Comment by unnonouno** [comment]
Please change file name (-03-15).

File: _posts/2017-11-16-Performance-of-LSTM-Using-CuDNN-v5.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+### Experimental Setting
+* Data: Random artificial sequence data (data size: 10,000)
+* Training epoch: 10
+* Comparing the average time per one epoch.
+* Performance time of
+  * forward time (for train data)
+  * forward time (for test data)
+  * backward time
+  * Default experiment setting:
+  * batchsize : 128
```
Comment (soskek) [inline _posts/2017-11-16-Performance-of-LSTM-Using-CuDNN-v5.md]: This list of settings (L122-127) looks further indented or marked in the original draft.

**Comment by unnonouno** [comment]
LGTM!

**Pull request merged**